docs/docs/index.html

```diff
@@ -115,38 +115,38 @@ <h3 class="title"><a href="index.html">OpenShift Pipelines</a></h3>
 <article class="doc">
 <h1 class="page">OpenShift Pipelines</h1>
 <div class="paragraph">
-<p>OpenShift Pipelines are cloud-native, continuous integration and delivery solutions based on Kubernetes resources. It uses <a href="https://tekton.dev">Tekton</a> Pipelines to automate deployments across multiple platforms by abstracting away the underlying details. Tekton introduces a number of standard Custom Resource Definitions (CRDs) for defining pipelines that are portable across Kubernetes distributions.</p>
+<p>OpenShift Pipelines is a cloud-native, continuous integration and delivery (CI/CD) solution based on Kubernetes resources. It uses <a href="https://tekton.dev">Tekton</a> Pipelines to automate deployments across multiple platforms by abstracting away the underlying details. Tekton introduces a number of standard Kubernetes <a href="https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/">custom resource definitions</a> (CRDs) for defining pipelines that are portable across Kubernetes distributions.</p>
 </div>
 <div class="ulist">
 <ul>
 <li>
-<p>OpenShift Pipelines are designed for microservices and decentralized teams</p>
+<p>OpenShift Pipelines is designed for microservices and decentralized teams</p>
 </li>
 <li>
-<p>They are standard CI/CD pipeline definitions that are easy to extend and integrate with existing tools, enabling you to scale on-demand</p>
+<p>OpenShift Pipelines uses standard CI/CD pipeline definitions that are easy to extend and integrate with existing Kubernetes tools, enabling you to scale on-demand</p>
```

Comment:
Added Kubernetes before tools to make the `enabling you to scale on-demand` portion more relevant